src/attrib.d

```diff
@@ -1294,34 +1294,39 @@ public:
         Dsymbol.setScope(sc);
     }
 
-    void compileIt(Scope* sc)
```

Comment:
I like this change, because it converts a side effect, setting `decls`, into a function return. This is better encapsulation.